packages/client/src/pages/register.vue

```diff
@@ -1,146 +1,233 @@
-              v-model:show="showPassword"
-              :rules="[requiredRule, validatePasswordRule]"
-              :label="t('auth.password')"
-              outlined
-              lazy-rules
-              autocomplete="password"
-              data-cy="password-field"
-            />
-            <password-strength-bar
-              :password-to-check="user.password"
-              :steps="STRENGTH_BAR_STEPS"
-            />
-            <k-password-input
-              v-model="user.passwordConfirmation"
-              v-model:show="showPassword"
-              :rules="[requiredRule, passwordMatchRule]"
-              :label="t('auth.confirmPassword')"
-              outlined
-              lazy-rules
-              autocomplete="password"
-              data-cy="password-confirmation-field"
-            />
-          </q-card-section>
-
-          <q-card-section>
-            <q-btn
-              class="full-width"
-              color="accent"
-              :label="t('auth.register')"
-              text-color="black-54"
-              type="submit"
-              :loading="isRegistering"
-              data-cy="submit-button"
-            />
-          </q-card-section>
-
-          <template v-if="SOCIAL_LOGIN_ENABLED">
-            <q-separator inset spaced />
-
-            <q-card-section class="column gap-8">
-              <span class="text-black-87">{{ t("common.or") }}</span>
-
-              <social-auth-buttons type="register" />
-            </q-card-section>
-          </template>
-
-          <q-separator inset spaced />
-
-          <q-card-section>
-            <span class="text-dark text-subtitle1">
-              {{ t("common.or") }}&nbsp;
-              <router-link to="login" class="text-dark">
-                {{ t("auth.login") }}
-              </router-link>
-            </span>
-          </q-card-section>
+  <q-page class="justify-center registration-page row">
+    <q-card class="column form-card gap-24 items-stretch no-wrap q-pa-lg">
+      <q-card-section class="column gap-24 no-padding no-wrap">
+        <q-img :src="theme.logo" fit="contain" height="60px" />
+
+        <q-form
+          class="column gap-8 items-stretch justify-center no-padding"
+          greedy
+          @submit="onSubmit"
+        >
+          <q-input
+            v-for="{ field, tooltip, ...inputData } in formData"
+            :key="field"
+            v-model="user[field]"
+            v-bind="inputData"
+            :autocomplete="
+              [
+                'password',
+                'passwordConfirmation',
+                'email',
+                'confirmEmail',
+              ].includes(field)
+                ? 'new-password'
+                : 'off'
+            "
+            bottom-slots
+            class="width-260"
+            lazy-rules
+            outlined
+          >
+            <template
+              v-if="
+                ['password', 'passwordConfirmation', 'delegate'].includes(field)
```

Comment:
Error message for invalid character occupies 3 lines and thus it can be hidden.
![image](https://github.com/ilmercatinodellibro/ilmercatinodellibro/assets/30588533/9ca0d268-2841-405f-baf8-fb583aeb4396) I think that if you can find a way to either:
1. shorten the message
2. enlarge the registration card to be like 360px 
3. or give more space between the two input fields the issue can be considered resolved.

Probably going with 1 or 2 is the best thing to do.

Reply:
Sadly had to resort to a combination of 1 and 3, as the shortest phrasing I could come up with still very much occupied more than 360px